packages/core/components/Filters.tsx

```diff
@@ -478,16 +478,23 @@ const Filters = (props: FilterProps) => {
   }
 
   if (visualizationConfig?.filters?.length === 0) return
-  const filterSectionClassList = [
-    `filters-section legend_${visualizationConfig?.legend?.hide ? 'hidden' : 'visible'}_${
-      visualizationConfig?.legend?.position || ''
-    }`,
-    type === 'map' ? general.headerColor : visualizationConfig?.visualizationType === 'Spark Line' ? null : theme
-  ]
+
+  const getCLasses = () => {
+    const classList = [
+      'filters-section',
+      type === 'map' ? general.headerColor : visualizationConfig?.visualizationType === 'Spark Line' ? null : theme
```

Comment:
return empty string here  so that null doesn't get interpolated into the classname